src/Symfony/Component/Yaml/Yaml.php

```diff
@@ -21,10 +21,11 @@
 class Yaml
 {
     /**
-     * Parses YAML into a PHP array.
+     * Parses YAML into a PHP array or an stdClass object.
```

Comment:
technically, it can also return a scalar. The Yaml format does not force to use a hash or a sequence on the top level AFAIK